<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>326</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>353</width>
    <height>326</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>353</width>
    <height>326</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dat Converter</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>30</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Action</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>331</width>
     <height>231</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>progress information</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Please input dat file path</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Please input tar  file path</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>datToPic()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>datToPic()</slot>
 </slots>
</ui>
